CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+Contributing to Search.gov
+=========================
+The search-gov repository includes the work of multiple contributors. You're encouraged to submit [pull requests](https://github.com/GSA/search-gov/pulls), [propose features and discuss issues](https://github.com/GSA/search-gov/issues).
+
+#### Contributing to Search.gov Multilingual Search Support
+Note: As of November 2021, management of multilingual support has moved from the [GSA/Punchcard](https://github.com/GSA/punchcard/) repository to the [GSA/search-gov](https://github.com/GSA/search-gov) repository.
+
+These files are in the YAML format.
```

Comment:
Sorry for requesting more changes! I just realized that we should clarify where those YAML files live, as it's less obvious than it was in Punchcard: https://github.com/GSA/search-gov/tree/master/config/locales

How about we change `These files are in the YAML format` to:

The Search.gov application uses the YAML files in [config/locales](https://github.com/GSA/search-gov/tree/master/config/locales) to provide localized translations of text strings based on the locale set for the user.